pom.xml

```diff
@@ -84,49 +84,6 @@
 
     <dependencies>
         <dependency>
-            <groupId>io.dropwizard.metrics</groupId>
-            <artifactId>metrics-core</artifactId>
-        </dependency>
```

Comment:
see them below actually. I'm wondering why we need these - Spring Boot has its own metrics out the box, i thought